libraries/AP_VisualOdom/AP_VisualOdom.h

```diff
@@ -75,6 +77,8 @@ class AP_VisualOdom
     // distances in meters, roll, pitch and yaw are in radians
     void handle_vision_position_estimate(uint64_t remote_time_us, uint32_t time_ms, float x, float y, float z, float roll, float pitch, float yaw, uint8_t reset_counter);
     void handle_vision_position_estimate(uint64_t remote_time_us, uint32_t time_ms, float x, float y, float z, const Quaternion &attitude, uint8_t reset_counter);
+    
+    void handle_vision_speed_estimate(uint64_t remote_time_us, uint32_t time_ms, const Vector3f &vel);
```

Comment:
we should add a comment above this although it could be as simple as, "// velocity in meters"